h2o-docs/src/product/data-science/algo-params/gainslift_bins.rst

```diff
@@ -0,0 +1,56 @@
+``gainslift_bins``
+------------------
+
+- Available in: GBM, DRF, Naïve-Bayes, XGBoost
```

Comment:
You need to link to this topic in each of the supported algo chapters (as in below)

<img width="798" alt="Screen Shot 2020-07-02 at 11 36 19 AM" src="https://user-images.githubusercontent.com/18428990/86397321-78ea8200-bc58-11ea-8d28-f527e9e4c1e7.png">